baremaps-postgres/src/main/java/com/baremaps/postgres/jdbi/BaseColumnMapper.java

```diff
@@ -0,0 +1,27 @@
+package com.baremaps.postgres.jdbi;
+
+import com.baremaps.osm.geometry.GeometryUtils;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import org.jdbi.v3.core.mapper.ColumnMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+import org.locationtech.jts.geom.Geometry;
+
+abstract class BaseColumnMapper<T extends Geometry> implements ColumnMapper<T> {
+
+  @Override
+  public T map(ResultSet r, int columnNumber, StatementContext ctx) throws SQLException {
+    byte[] bytes = hexStringToByteArray(r.getString(columnNumber));
```

Comment:
what about `r.getBytes()` ?

Reply:
Unfortunately, getBytes return the bytes of a string. This is due to the fact that the postgresql drive uses text to transport data.